src/Contexts/PlaylistContext/index.jsx

```diff
@@ -17,6 +21,116 @@ const PlaylistProvider = ({ children }) => {
     watchLaterPlaylist:[],
     historyPlaylist:[]
   });
+  const {LOADING_DATA,LOAD_DATA,UPDATE_LIKED_VIDEO,UPDATE_HISTORY,UPDATE_WATCH_LATER,CREATE_PLAYLIST_WITH_VIDEO} = dispatchActioTypes
+
+  useEffect(()=>{
+    if(!user.isAuthenticated){
+      setShowPlaylistCreationModal(false);
+    }
+  },[user.isAuthenticated])
+  useEffect(()=>{
+    if(user.isAuthenticated){
+      (async () => {
+        userVideoDispatch({
+          type: LOADING_DATA,
+          payload: { status: true },
+        });
+        try{
+          const res = await axios.get("/api/user/history", {
+            headers: { authorization: user.encodedToken },
+          });
+          if (res.status === 200) {
+            userVideoDispatch({
+              type: UPDATE_HISTORY,
+              payload: { data:res.data },
+            });
+          }
+        }
+        catch(err){
+          console.log("initial hitory data fetch error",err)
```

Comment:
suggestion: remove `console.log`

Reply:
yes . Will add toasts to make user experience more better